web-gateway/app/views/item.scala.html

```diff
@@ -20,6 +20,7 @@
 
     <div >
         <h1>@item.getItemData.getTitle</h1>
+        <p>@message("instruction.startAuction")</p>
```

Comment:
I created an item as user `bob` and started the auction. When I went to place a bid as `Alice` I could see this message.

![screen_shot_2017-03-17_at_11_05_31](https://cloud.githubusercontent.com/assets/762126/24038539/21dacfd4-0b02-11e7-9a0b-0db51af6cb12.png)